apps/ideas/forms.py

```diff
@@ -58,7 +71,14 @@ class PartnersSectionForm(BaseForm):
 
     class Meta:
         model = AbstractPartnersSection
-        exclude = []
+        fields = list(
+            chain.from_iterable((
```

Comment:
importing chain to write three lines less code might be overdoing it? explicit and so on?